quarto/src/Repository/GameRepository.php

```diff
@@ -26,7 +26,12 @@ public function getOpenedGamesList(array $tokenList)
         if (count($tokenList) === 0) {
             $tokenList=["NOTHING"];
         }
-        $query = $this->em->createQuery("SELECT g FROM App:Game g  " .
+        $query = $this->em->createQuery("SELECT g.id_game idGame,
+            g.number_players numberPlayers,
+            g.solo_game soloGame,
+            g.closed,
+            '' token
```

Comment:
??

Reply:
Returns structure have to be identical in all lists requests, so they can be managed the same way by clients.
For openened games list, the user have no token (because he is not registerd in the game)
So I return an empty field for token